workspaces/services/devtestlabs/.env.sample

```diff
@@ -0,0 +1,9 @@
+TRE_ID=__CHANGE_ME__
+WORKSPACE_ID=__CHANGE_ME__
+AZURE_TENANT_ID=_CHANGEME_
```

Comment:
Weren't the name standard ATM_* in the other bundle?

Reply:
ARM_, yup, let me investigate.